plugin/metrics/factory.go

```diff
@@ -33,6 +34,7 @@ const (
 	disabledStorageType = ""
 
 	prometheusStorageType = "prometheus"
+	grpcStorageType       = "grpc-plugin"
```

Comment:
So we have `--metrics-storage-type=grpc-plugin` but then factory flags under `--grpc-metrics.*`? I can't think if a better name, but at least we should keep these in sync, so let's go with `grpc-metrics`